_scripts/instruction-widget/templates/install/commonauto.html

```diff
@@ -1,4 +1,6 @@
 {{#jessie}}
+<p>If you would like to help test a new way to install Certbot that automatically renews your certificates and stays up-to-date, please choose "snapd" in the system dropdown above.</p>
```

Comment:
We don't want to include this for Debian Jessie. They're unable to use the snaps unfortunately.

I think you can just delete the change here.

Reply:
Oops, misunderstood, my bad.